src/tike/view.py

```diff
@@ -108,6 +108,9 @@ def _confidence_ellipse(x, y, ax, n_std=3.0, facecolor='none', **kwargs):
         raise ValueError("x and y must be the same size")
 
     cov = np.cov(x, y)
+    if np.all(cov == 0):
```

Comment:
Should this be something like abs(cov) < eps?

Reply:
```suggestion
    if np.all(np.abs(cov) < 1e-6):
```